test_cli.py

```diff
@@ -924,7 +975,9 @@ def test(ctx):
     # Modify to non-ascii tags
     # TODO: enable with Unicode support.
     non_ascii_tags = ['你好世界😊', 'fáncy ünicode']
-    run_command([cl, 'wedit', wname, '--tags'] + non_ascii_tags, 1)
+    run_command(
+        [cl, 'wedit', wname, '--tags'] + non_ascii_tags, 1, force_subprocess=True
+    )  # TODO: find a way to make this work without force_subprocess
```

Comment:
This needs to eventually be fixed -- for some reason this command only succeeds when using `force_subprocess`.